api/client/commands.go

```diff
@@ -1191,10 +1190,6 @@ func (cli *DockerCli) CmdPush(args ...string) error {
 	name := cmd.Arg(0)
 
 	cli.LoadConfigFile()
```

Comment:
The LoadConfigFile is used for authentication credentials, was not used for the key